packages/protobuf/src/next/to-json.ts

```diff
@@ -0,0 +1,486 @@
+// Copyright 2021-2024 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import type { DescEnum, DescField } from "../descriptor-set.js";
+import {
+  FeatureSet_FieldPresence,
+  FieldDescriptorProto_Label,
+} from "../google/protobuf/descriptor_pb.js";
+import type { JsonValue, JsonObject } from "../json-format.js";
+import { assert } from "../private/assert.js";
+import { protoBase64 } from "../proto-base64.js";
+import { protoCamelCase } from "./reflect/names.js";
+import { reflect } from "./reflect/reflect.js";
+import { ScalarType } from "./reflect/scalar.js";
+import type { ReflectMessage } from "./reflect/reflect.js";
+import type { Message } from "./types.js";
+import type {
+  Any,
+  Duration,
+  FieldMask,
+  ListValue,
+  Struct,
+  Timestamp,
+  Value,
+} from "./wkt/index.js";
+import { anyUnpack } from "./wkt/index.js";
+import { isWktWrapperDesc } from "./reflect/wkt.js";
```

Comment:
We have WKT-specific code here, in wkt/, and in reflect/.

I don't think we should everything over to wkt/ (import cycle on `JsonWriteOptions`, still need `"google.protobuf.NullValue"` here, and more exports to deal with). 

But I think it makes sense to move reflect/wkt.ts to wkt/wrappers.ts.

Not in this PR, just thinking out load. If you want to do it, go for it. If you disagree, let me know.